justfile

```diff
@@ -125,20 +125,15 @@ test *args:
 black *args=".": devenv
     $BIN/black --check {{ args }}
 
-isort *args=".": devenv
-    $BIN/isort --check-only --diff {{ args }}
-
-flake8 *args="": devenv
-    $BIN/flake8 {{ args }}
+django-upgrade *args="$(find applications jobserver services tests -name '*.py' -type f)": devenv
```

Comment:
How come the list of django apps has changed here?

Reply:
Bad rebase when I pulled the update out into another PR, fixed!

Reply:
And I managed to botch fixing this here 🤦‍♂️ PR incoming!